### DISCUSSION TITLE
articles/learning-english

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#134] [Comments]
http://hutusi.com/articles/learning-english

 有同事向我请教英语学习的方法，我很诧异。因为我英语水平其实还很差，只不过在公司内部的英语培训上跟外教分享了自己“苦学”的历程，让外教很惊讶，他也向这位同事提到了我。

**Comment by apert4160** [comment]
第七个段落有个typo：
>最开始的时候，是从背单词开始。因为底子薄，一定要掌握一定的词汇量才能够去联系听说读写。

[联系]应该是练习？

**Comment by hutusi** [comment]
> @apert4160
> 第七个段落有个typo：
> >最开始的时候，是从背单词开始。因为底子薄，一定要掌握一定的词汇量才能够去联系听说读写。
> 
> [联系]应该是练习？

谢谢，是的。

**Comment by Junyi-99** [comment]
确实，看剧在精不在多

**Comment by hutusi** [comment]
> @Military-Doctor
> 确实，看剧在精不在多

回看五年前自己写的这篇文章，这五年英语却没什么进步，惭愧...
